README.md

```diff
@@ -1186,6 +1187,46 @@ class IsTallMeta extends MetaBase {
 }
 ```
 
+#### DomWrapper
+
+`DomWrapper` is used to wrap DOM that is created _outside_ of the virtual DOM system.  This is the main mechanism to integrate _foreign_ components or widgets into the virtual DOM system.
+
+The `DomWrapper` generates a class/constructor function that is then used as a widget class in the virtual DOM.  `DomWrapper` takes up to two arguments.  The first argument is the DOM node that it is wrapping.  The second is an optional set of options.
```

Comment:
should we mention that the component that `DomWrapper` produces allows you to pass `VirtualDomProperties` just like you would use on a normal `v` call?